sellercenter-setting/src/main/java/com/gfg/sellercenter/setting/service/SettingServiceInterface.java

```diff
@@ -0,0 +1,10 @@
+package com.gfg.sellercenter.setting.service;
+
+import com.gfg.sellercenter.setting.entity.Setting;
+
+public interface SettingServiceInterface {
+
+    Setting getSetting(String field, int sellerId) throws Exception;
```

Comment:
It seems that now those method are not throwing exception. If so, this can be removed from declaration.

Reply:
now they are throwing exceptions